<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-8LD3H0DF" failures="1" tests="9" name="admin.UserController" time="1.222" errors="0" timestamp="2025-03-27T09:30:38 IST" skipped="4">
  <testcase classname="admin.UserController" name="test08GetSupplierDetailsByAuctionId" time="0.182"/>
  <system-out/>
  <testcase classname="admin.UserController" name="test10GetInactiveUsers" time="0.091"/>
  <system-out/>
  <testcase classname="admin.UserController" name="test01GetUserDetails" time="0.022"/>
  <system-out/>
  <testcase classname="admin.UserController" name="UpdateUserUpload" time="0.904">
    <failure message="Expected 200 OK or 201 Created, but got: 500 expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected 200 OK or 201 Created, but got: 500 expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at admin.UserController.UpdateUserUpload(UserController.java:99)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- UpdateUserUpload -->
  <system-out/>
  <testcase classname="admin.UserController" name="test09GetAppliedUsersKyc" time="0.023"/>
  <system-out/>
  <testcase classname="admin.UserController" name="test11GetAllSuppliers">
    <skipped/>
  </testcase> <!-- test11GetAllSuppliers -->
  <system-out/>
  <testcase classname="admin.UserController" name="test06GetUserSupplierDetails">
    <skipped/>
  </testcase> <!-- test06GetUserSupplierDetails -->
  <system-out/>
  <testcase classname="admin.UserController" name="test07GetKycByUserId">
    <skipped/>
  </testcase> <!-- test07GetKycByUserId -->
  <system-out/>
  <testcase classname="admin.UserController" name="test05UpdateUploadedUsers">
    <skipped/>
  </testcase> <!-- test05UpdateUploadedUsers -->
  <system-out/>
</testsuite> <!-- admin.UserController -->
